anvill/python/anvill/program.py

```diff
@@ -175,8 +200,19 @@ def proto(self):
             proto["functions"].append(func.proto())
 
         for source, target in self._control_flow_redirections.items():
+            # Use 2-entry lists so that we don't use 'source' as a key. That
+            # would turn it into a string in the final JSON forcing us to
+            # handle integers in two different ways
             proto["control_flow_redirections"].append([source, target])
 
+        for entry in self._ctrl_flow_targets.values():
```

Comment:
`_control_flow_targets`, for consistency with `_control_flow_redirections`. We're not paying for characters :-P